numpy/core/_add_newdocs_scalars.py

```diff
@@ -49,14 +50,16 @@ def type_aliases_gen():
     ])
 
 
+
+
 def add_newdoc_for_scalar_type(obj, fixed_aliases, doc):
     # note: `:field: value` is rST syntax which renders as field lists.
     o = getattr(_numerictypes, obj)
 
     character_code = dtype(o).char
     canonical_name_doc = "" if obj == o.__name__ else ":Canonical name: `numpy.{}`\n    ".format(obj)
     alias_doc = ''.join(":Alias: `numpy.{}`\n    ".format(alias) for alias in fixed_aliases)
-    alias_doc += ''.join(":Alias on this platform: `numpy.{}`: {}.\n    ".format(alias, doc)
+    alias_doc += ''.join(":Alias on this platform ({} {}): `numpy.{}`: {}.\n    ".format(platform.system(), platform.machine(), alias, doc)
```

Comment:
Is `system` + `machine` better or worse than `platform()`?

(I have no idea)

Reply:
From what I can tell `platform.system` is pretty much the same as `sys.platform`. I threw in the `platform.machine` because I would expect what numeric types would vary on the same OS between say, the 64 bit and 32 bit version. For example on my machine `system` + `machine` is `Linux x86_64`, and 'platform` is `Linux`.

Reply:
sorry, by `platform` I meant `platform.platform()`

Reply:
`platform.platform` has a ton of information (`Linux-4.19.128-microsoft-standard-x86_64-with-glibc2.29` on my WSL2 setup), to the point where I think it would probably clutter up the documentation more than it would help if it were included more than once. Maybe have the full platform be somewhere on that page and then link back to it whenever 'this platform' is mentioned?

Reply:
agreed, this looks good as is.